src/controllers/connectionConfig.ts

```diff
@@ -0,0 +1,67 @@
+import { rejects } from "assert";
+import { resolve } from "path";
+
+const Connection = require('tedious').Connection;
+const RequestStatement = require('tedious').Request;
+let connection;
+
+export function setupConnection(portVal){
+    const config = {
+        "server": "localhost",
+        "authentication": {
+          "type": "default",
+          "options": {
+            "userName": "DanLeb",

```

Comment:
This is fine for now, but I'll chat about environment variables this afternoon and we can set it up more conventionally.